### ISSUE TITLE
Add option for simplified output of restore utility

### ISSUE THREAD

**Issue by gamename** [rethinkdb/rethinkdb#5378]
Slack conversation on the RethinkDB channel:

```
i guys.  Not an emergency, but is there some way to disable the % scale output of the rethink restore util?  I’m calling the utility from a script and the output is huge (see attached)
Or rather, the output makes the wrapper script’s logs huge.

new messages
neilhanlon [5:59 PM] 
>/dev/null
?
er...
>/dev/null
redirect stdio to /dev/null

tennis [6:12 PM] 
@neilhanlon: sure, but that leaves me completely blind.  I’m thinking of some compromise that will be informative, but not ascii art.

neilhanlon [6:19 PM] 
Only thing I can think of (besides actually modifying the code) is to pipe it into tr or something to replace the carriage return with a newline
https://github.com/rethinkdb/rethinkdb/blob/next/drivers/python/rethinkdb/_import.py#L601
 
GitHub
rethinkdb/rethinkdb
rethinkdb - The open-source database for the realtime web.

tennis [6:20 PM] 
ah, ok.  That’s a good compromise.  Thanks.

mglukhov [6:28 PM] 
@tennis: could you open an issue describing how you produced that?

[6:28] 
we can explore options there for how to improve the output, or provide a flag for suppressing it

tennis [6:29 PM] 
@mglukhov: Sure. Np. Thanks for working w/me on that.
```

**Comment by gamename** [comment]
![pasted image at 2016_02_08 05_46 pm](https://cloud.githubusercontent.com/assets/42153/12904140/b99bbae8-ce92-11e5-83aa-aa8adfbce9b0.png)

**Comment by gamename** [comment]
![pasted image at 2016_02_08 05_46 pm](https://cloud.githubusercontent.com/assets/42153/12904149/d0ccc9b4-ce92-11e5-8516-7f22b2f06672.png)

**Comment by gamename** [comment]
The script which uses the restore util is a simple bash script:

```
...
if rethinkdb restore "${LOCAL_COPY}" \
   --shards "${CLUSTER_SIZE}"        \
   --replicas "${CLUSTER_SIZE}"      \
   --force; then
  log "${LOCAL_COPY} restoral Succeeded"
  rc=0
else
  log "${LOCAL_COPY} restoral FAILED"
fi
...
```

**Comment by danielmewes** [comment]
So basically an option to disable the progress bar?

That should be pretty easy, and sounds useful for scripting purposes.

**Comment by danielmewes** [comment]
I'm going to put this into `backlog` for now, since it's not a top priority. But we'll keep it in mind (alternatively pull requests are welcome if anyone wants to pick this up!).

**Comment by NeilHanlon** [comment]
I think this is resolved by #5528  -- @danielmewes ?

**Comment by danielmewes** [comment]
Indeed, thanks for the note @NeilHanlon .
There is now a `--quiet` option added by @nh2.
